ecommerce_connector/models/ecommerce_connector.py

```diff
@@ -560,6 +560,8 @@ def _get_credit_note_lines(self, lines, ecommerce_connection):
     def _check_general_mandatory_fields(
         self, errors, values, company_id, ecommerce_connection, excluded_fields=False
     ):
+        if not excluded_fields:
+            excluded_fields = []
```

Comment:
Why don't you make the default parameter =[] in the function declaration? You save the if's

Reply:
@ValentinVinagre Because in python, using [] or {} as default parameters in functions causes problems. The precommit also tells you to not do that